api/MediaList.json

```diff
@@ -196,10 +196,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaList/length",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "44"
```

Comment:
I doubt this is right. http://mdn-bcd-collector.appspot.com/tests/api/MediaList doesn't involve an instance, and `length` is even more special than some other attributes.

A custom test could probably be added based on https://developer.mozilla.org/en-US/docs/Web/API/MediaList#Examples